inference-engine/src/transformations/src/transformations/op_conversions/convert_divide.cpp

```diff
@@ -26,25 +27,33 @@ bool convert_divide(std::shared_ptr<ngraph::Node> node) {
         return false;
     }
 
-    ngraph::Output<ngraph::Node> pow = std::make_shared<ngraph::opset1::Power>(div->input_value(1),
+    std::shared_ptr<ngraph::Node> pow = std::make_shared<ngraph::opset1::Power>(div->input_value(1),
                                                        ngraph::op::Constant::create(div->get_input_element_type(1), ngraph::Shape{}, {-1}));
 
     if (std::dynamic_pointer_cast<ngraph::op::Constant>(div->get_input_node_shared_ptr(1))) {
         if (auto const_pow = ngraph::get_constant_from_source(pow)) {
             pow = const_pow;
         } else {
-            NGRAPH_DEBUG << "ConvertDivide has failed due to unsupported evaluate type in " << pow.get_node();
+            NGRAPH_DEBUG << "ConvertDivide has failed due to unsupported evaluate type in " << pow.get();
             return false;
         }
     } else {
-        ngraph::copy_runtime_info(div, pow.get_node_shared_ptr());
+        ngraph::copy_runtime_info(div, pow);
```

Comment:
Looks like a duplicate of https://github.com/openvinotoolkit/openvino/pull/8736/files#diff-17721168d1c4ff54c1856f328053bf032f14bd58b1b0566923d36deccbb9cff9R49
Let's keep one.

Reply:
This PR is closed. Let's do the review here https://github.com/openvinotoolkit/openvino/pull/8840